src/Codeception/Step/Executor.php

```diff
@@ -1,15 +1,21 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\Step;
 
+use Closure;
 use Codeception\Step as CodeceptionStep;
 use Codeception\Lib\ModuleContainer;
 
 class Executor extends CodeceptionStep
 {
+    /**
+     * @var Closure|null
+     */
+    protected $callable;
```

Comment:
I think that this field can be typed as Closure, because it is assigned in constructor.